src/Apps/Order/Routes/Shipping/index.tsx

```diff
@@ -790,8 +790,9 @@ export const ShippingRoute: FC<ShippingProps> = props => {
             <Collapse open={showArtsyShipping}>
               <Text variant="sm">Artsy shipping options</Text>
               <Text variant="xs" mb="1" color="black60">
-                All options are eligible for Artsy’s Buyer Protection policy,
-                which protects against damage and loss.
+                Please note that these are estimates and may change once offer
```

Comment:
should we check that the order is in offer mode here?

Reply:
Great catch there 🦅

Reply:
Added conditional logic for this help text

Buy Now experience:
<img width="800" alt="Screen Shot 2023-05-26 at 8 58 47 AM" src="https://github.com/artsy/force/assets/12748344/ceb79920-2959-4851-9d76-9eff52b5caea">

Make Offer experience:

<img width="800" alt="Screen Shot 2023-05-26 at 8 57 38 AM" src="https://github.com/artsy/force/assets/12748344/012493cf-8442-4cb6-ad9b-3331a0a275b6">